app/src/main/java/org/dicekeys/app/fragments/AssembleFragment.kt

```diff
@@ -57,7 +52,7 @@ class AssembleFragment : AppFragment<AssembleFragmentBinding>(R.layout.assemble_
                 FaceRead.diceKeyFromJsonFacesRead(facesReadJson)?.let { diceKeyFaceRead ->
 
                     // Convert to Face
-                    viewModel.setDiceKey(DiceKey(faces = diceKeyFaceRead.faces.map {
+                    assembleViewModel.setDiceKey(DiceKey(faces = diceKeyFaceRead.faces.map {
```

Comment:
Does the DiceKey in assembleViewModel get automatically erased when the assembly flow closes?

Reply:
Is garbaged collected when assembleViewModel gets destroyed.

Reply:
Right, but does the viewModel get destroyed the moment you navigate away from the view, or is it a persistent model that gets updated if you go back?